optimism.tokenlist.json

```diff
@@ -1067,6 +1067,61 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 10,
+      "address": "0x3bb4445d30ac020a84c1b5a8a2c6248ebc9779d0",
+      "name": "Theranos Coin",
+      "symbol": "LIZ",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/LIZ.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
```

Comment:
Copy-paste gone wrong? We already have Theranos Coin in the token list.

Reply:
F, fixing

Reply:
all done